packages/cpp/src/node/debug/cpp-debug-contribution.ts

```diff
@@ -0,0 +1,153 @@
+/********************************************************************************
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+const path = require('path');
+const packageJson = require('../../../package.json');
+const debugAdapterDir = packageJson['debugAdapter']['dir'];
+
+import { injectable } from 'inversify';
+import { DebugConfiguration } from '@theia/debug/lib/common/debug-common';
+import { DebugAdapterContribution, DebugAdapterExecutable } from '@theia/debug/lib/node/debug-model';
+import * as Ajv from 'ajv';
```

Comment:
We are using `json-schema-defaults` similar to @jonahgraham , I wonder if there is functional crossover between the packages?

Reply:
It seems like it: https://github.com/epoberezkin/ajv#assigning-defaults

But `ajv` was already used in `core` and `keymaps` packages, this is why I used it here.

Reply:
nice, i'll see if I can use this instead :)